src/test/java/org/sonar/plugins/buildbreaker/QualityGateBreakerTest.java

```diff
@@ -89,9 +101,32 @@ public void testShouldExecuteDisabledFromSkipSetting() {
     assertEquals(false, new QualityGateBreaker(fileSystem, config).shouldExecuteOnProject());
   }
 
+  @Test
+  public void testLoadReportTaskTxtFile() {
+    Settings settings = new MapSettings();
+    Configuration config = new ConfigurationBridge(settings);
+
+    Properties reportTaskProps = new QualityGateBreaker(fileSystem, config).loadReportTaskProps();
+    assertNotNull(reportTaskProps);
```

Comment:
What do you think of using  Design by Contract approach like [lombok's `NonNull`](https://stackoverflow.com/a/51523435/11475085)? Maybe time to become a Kotlin fan (just kidding). :)

Reply:
Hm, at the very least it actually makes sense to mark loadReportTaskProps as `@NotNull`, because it really can't return null. So the `assertNotNull` checks here are really superfluous :) Will change.